### ISSUE TITLE
Message to SUMO is lost

### ISSUE THREAD

**Issue by yunpangfloetteroed** [DRIVER-EU/python-test-bed-adapter#1]
We sent several messages from test bed to SUMO, but the first was not received by SUMO, when we set  the option reset_offset_on_start = False. If we set it with TRUE, the second message was not received by SUMO.

**Comment by HugoJBello** [comment]
I do not fully understand the problem. But if you set reset_offset_on_start, you will only recieve the messages sent from the moment you start listening (that is from the moment you run the consumer.
Also try setting the option offset_type = "EARLIEST" to catch all messages instead of the last ones.

**Comment by yunpangfloetteroed** [comment]
Yes, we run the consumer first, then run the producer. We also tried the option "EARLIST". But the second message was still not received by SUMO.

**Comment by yunpangfloetteroed** [comment]
The second message can meanwhile received by SUMO with use of the option EARLIST. But, the processing is not totally correct.... We will check the reason.

**Comment by HugoJBello** [comment]
Ok thanks for the question. I think I will set offset_type = "EARLIEST" as default, because it makes more sense. 
If you pass me the script of your consumer I can try to take a look at it to see if it has to do with the python-test-bed-adapter options.

**Comment by yunpangfloetteroed** [comment]
It will be great! :) You can find the script sumo_connector.py as consumer and the script test_sumo_connector.py as producer under https://github.com/DRIVER-EU/sumo-connector .

**Comment by HugoJBello** [comment]
I have taken a look to the script summo_connector.py, I do not see anything wrong with the test bed options. Could it be something related with the function    ` def addToQueue(self, message):`? maybe there is not decoded message for some of the messages and we are not adding them.
